package.json

```diff
@@ -43,8 +43,8 @@
     "@types/select2": "^4.0.48",
     "@types/selenium-webdriver": "^4.0.0",
     "@types/wavesurfer.js": "^3.3.1",
-    "@typescript-eslint/eslint-plugin": "^3.0.0",
-    "@typescript-eslint/parser": "^3.0.0",
+    "@typescript-eslint/eslint-plugin": "^4.0.0",
```

Comment:
This looks like a major update, make sure to check whether it's working as expected!